.github/workflows/dev-aws-ecr.yml

```diff
@@ -35,17 +35,16 @@ jobs:
 
       run: |
         echo $CERT > saml.pem
-        docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
+        docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG . --build-arg BUILD=$IMAGE_TAG
```

Comment:
In addition to making this change here, you'll also want to make it here https://github.com/USSF-ORBIT/ussf-portal-client/blob/main/.github/workflows/build-push-c1-art.yml (might need to merge main into your branch as this was merged this morning).

Reply:
ahh thank you! added 🌟